joomla/joomla-cms
# [4.1] New Quickicon Tags

Pull Request for Issue # .
A user who works much with tags suggested this new Icon. It could be added with default = hide, so no database change is needed. 

### Summary of Changes
New Icon "Tags" for the Quickicon site Panel. 

<img width="931" alt="quicicon-tags" src="https://user-images.githubusercontent.com/1035262/106283560-62ce8f80-6242-11eb-8188-4404bee9fd4b.PNG">

### Testing Instructions
Open the module quickicon, activate the tag icon. 
Got to dashboard and see the tags Quickicon.
The number of published tags will be shown if the count option is selected. 
Klick on the big button and the tags overview is displayed.
Klick on the + icon and the tags edit view is desplayed.

### Actual result BEFORE applying this Pull Request
No Quickicon for tags


### Expected result AFTER applying this Pull Request
The tag icon is correctly displayed


### Documentation Changes Required
yes
